pkgs/applications/science/logic/ls4/default.nix

```diff
@@ -0,0 +1,38 @@
+{ zlib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "ls4-${version}";
+
+  version = "1.0";
+
+  buildInputs = [ zlib ];
+
+  buildPhase = ''
+  cd core
+  # These object files were committed into the git repo, perhaps accidentally
+  rm -f aiger.o
+  rm -f aiger.o_32
+  gcc -g -O3 -c aiger.c
+  make
+  '';
+
+  installPhase = ''
+  mkdir -p $out/bin
+  cp ls4 $out/bin
```

Comment:
``install -D ls4 $out/bin/ls4`` or similar can be used to replace these two lines.

Reply:
Done.